py/build.gradle

```diff
@@ -111,8 +107,8 @@ task distPython(type: Exec, dependsOn: checkPythonEnv) {
         }
 
         copy {
-            from "${configurations.compile.join(',')}"
```

Comment:
me to, but why I decided to do this: Because we can depend on specific configuration anymore, we depend on all artefacts produced by `:sparkling-water-assembly`. I can use them ( there's around 190 artefacts ) and filter for sparkling-water assembly jar. It seems cleaner to me to declare it like this then filtering almost 200 hundred artefacts. But I can the the filtering, it actually worked for me too.